interface/wx/listbox.h

```diff
@@ -321,6 +318,20 @@ class wxListBox : public wxControl,
     */
     int GetTopItem() const;
 
+    /**
+        MSW-specific function for setting custom tab stop distances.
+
+        @param tabStops
+            Cumulative tab distances array measured in "quarters of the average character width for the font that is selected into the list box."
+
+        This function is not available in the other ports by design, any
+        occurrences of it in the portable code must be guarded by
+        @code #ifdef __WXMSW__ @endcode preprocessor guards.
+
```

Comment:
Shouldn't the `@since 3.1.4` tag be here?

Not sure if the instructions about guarding MSW-specific code are really necessary...

Reply:
> Shouldn't the @since 3.1.4 tag be here?
I also wondered about the @since tag in a comment above, not knowing who/when adds it.
> Not sure if the instructions about guarding MSW-specific code are really necessary...
I got inspired (copy-pasted) from docs of other MSW-specific methods. Will gladly remove it, just please confirm.